operator/test/api/metrics/v1alpha3/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
are the files in this dir generated automagically or we need to keep them aligned via c/p?

Reply:
everything in test/api/metrics/* folder we need to keep aligned with the `metrics-operator`. I was trying to make it work automatically with generating CRDs and manifests, but we do not need these parts and it will require more entries in the kubebuilder framework (in PROJECT file etc). What we now have is the bare minimum to make `KeptnMetric` usable for our tests. Additionally to make these files ignored from re-generation I needed to update the `make manifests` and `make generate` commnads